helm/cron_jobs/replayer-cronjobs/templates/cronjobs.yaml

```diff
@@ -0,0 +1,60 @@
+{{- $rootContext := . }}
+{{- range $replayer := .Values.replayers }}
+{{- with $replayer -}}
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "replayer-cronjobs.fullname" $rootContext }}-{{ .name }}
```

Comment:
Kubernetes is complaining that this is too long for a given Release name. Can we change this line to:
` name: mina-cronjob-{{ .name }}`